content/docs/typechecking-with-proptypes.md

```diff
@@ -196,3 +196,47 @@ class Greeting extends React.Component {
 ```
 
 The `defaultProps` will be used to ensure that `this.props.name` will have a value if it was not specified by the parent component. The `propTypes` typechecking happens after `defaultProps` are resolved, so typechecking will also apply to the `defaultProps`.
+
+### Functional Components
+
+If you are using Functional Components in your regular development, you may want to make some small changes to allow PropTypes to be proper applied.
+
+Let's say you have a simple component like this:
+
+```javascript
+export default function ({ name }) {
```

Comment:
Could you give this a name? React is discouraging anonymous functions since they appear as "unknown" in component stack traces.